Directory.Packages.props

```diff
@@ -74,6 +74,7 @@
     <PackageVersion Include="MySqlConnector.DependencyInjection" Version="2.3.1" />
     <PackageVersion Include="Npgsql.DependencyInjection" Version="8.0.0-rc.2" />
     <PackageVersion Include="Npgsql.EntityFrameworkCore.PostgreSQL" Version="8.0.0-rc.2" />
+    <PackageVersion Include="Oracle.ManagedDataAccess.Core" Version="23.3.0-dev" />
```

Comment:
We should be depending on stable versions, if possible. Is a stable version going to ship soon?

Reply:
This is the version that supports Open Telemetry. My intention is to evolve this PR as they develop the stable version.

Reply:
Perfect. That makes sense to me.